docs/docs/validation-action.mdx

```diff
@@ -0,0 +1,248 @@
+---
+id: validation-action
+sidebar_label: Slot Validation Actions
+title: Slot Validation Actions
+abstract: Learn how `ValidationAction` class is implemented in the Rasa SDK.
+---
+
+There are two helper classes in Rasa SDK with the role of executing custom slot extraction and validation:
+- `ValidationAction`: the base class for custom actions specialized on custom extraction and / or validation
+of slots that can be set or updated outside of a form context.
+- `FormValidationAction`: the base class for custom actions extracting and validating slots that are set only within
+the context of a form.
+
+## `ValidationAction` class implementation
```

Comment:
I would suggest organizing this as:
```
Intro including ## How to subclass slot validation actions content
ValidationAction`
     Usage
     Implementation
FormValidationAction
      Usage
      Implementation
   ```
   
   Users will want usage before implementation details, and the classes group naturally to themselves